docs/Developers/Investments/SmartContracts/Insurance/Buying.md

```diff
@@ -17,4 +22,6 @@ function getReceivedInsurance(
 ```
 
 - ***deposit*** - the amount of tokens to be deposited
-- **returns** -> the received insurance tokens
\ No newline at end of file
+- **returns** -> the received insurance tokens
+
+⚠️⚠️ The ***`proposeClaim()`*** function is the ***`createProposal()`*** function on **DeXe DAO**.
```

Comment:
The insurance claim proposal is done through DEXE DAO proposal creation. The community will decide whether to proceed with the payout or not.